test/layout.js

```diff
@@ -87,4 +87,28 @@ test('orders weeks correctly with weeks spanning years', t => {
   const firstDay = firstWeek.find('.tt-cal-day').first();
 
   t.is(firstDay.text(), '20171224');
-})
+});
+
+test('inserts dummy days when day tz offset is less than the local tz', t => {
+  const localOffset = moment().utcOffset();
+  const firstDay = moment('2017-05-30').startOf('day').utcOffset(localOffset - 60, true);
+  const lastDay = firstDay.clone().add(20, 'days');
+
+  // Do a render that we won't use, first, to make sure that we're testing
+  // against over-eager memoization.
+  render(<TTReactCalendar
+    compactMonths={true}
+    firstRenderedDay={firstDay.clone().utcOffset(localOffset, true)}
+    lastRenderedDay={lastDay.clone().utcOffset(localOffset, true)}
```

Comment:
Btw, passing that `true` flag to `utcOffset` makes it keep the local time constant, instead of the actual time. For instance, `moment('01-01-1970').utcOffset(-5, true)` would give you `1970-01-01T00:00:00-05:00`.